To github.com:SE4Sci-S2024/git-exercise-labree-tyler.git
 ! [rejected]        main -> main (fetch first)
error: failed to push some refs to 'github.com:SE4Sci-S2024/git-exercise-labree-tyler.git'
hint: Updates were rejected because the remote contains work that you do not
hint: have locally. This is usually caused by another repository pushing to
hint: the same ref. If you want to integrate the remote changes, use
hint: 'git pull' before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
----
hint: You have divergent branches and need to specify how to reconcile them.
hint: You can do so by running one of the following commands sometime before
hint: your next pull:
hint: 
hint:   git config pull.rebase false  # merge
hint:   git config pull.rebase true   # rebase
hint:   git config pull.ff only       # fast-forward only
hint: 
hint: You can replace "git config" with "git config --global" to set a default
hint: preference for all repositories. You can also pass --rebase, --no-rebase,
hint: or --ff-only on the command line to override the configured default per
hint: invocation.
fatal: Need to specify how to reconcile divergent branches.
----
# This is the experimental source code file
a = 1 + 2.5
b = -10
print(a)
print(b)
----
interactive rebase in progress; onto d95b802
Last command done (1 command done):
   pick d51bedf Local commit
No commands remaining.
You are currently rebasing branch 'main' on 'd95b802'.
  (all conflicts fixed: run "git rebase --continue")

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	modified:   source.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	p3_log

